pkg/console/assets/bindata.go

```diff
@@ -970,11 +970,13 @@ var _bindata = map[string]func() (*asset, error){
 // directory embedded in the file by go-bindata.
 // For example if you run go-bindata on data/... and data contains the
 // following hierarchy:
-//     data/
-//       foo.txt
-//       img/
-//         a.png
-//         b.png
+//
```

Comment:
not sure why this is happening but these changes are not allowed by linter.
please remove them